README.md

```diff
@@ -1,26 +1,270 @@
+
+These instructions should work for both the RPi 3 and 4. You are free to use other versions of RPi, ROS, or OS, but setting these up is not covered here and it is not guaranteed that those will work.
+
+### 2.1 Installing the operating system
+
+The first step is to install the Ubuntu (Recommended), or Raspbian OS on your Raspberry Pi.
+
+#### 2.1.1 Ubuntu 18.04 (recommended)
+
+Download Ubuntu 18.04 from [here](https://ubuntu.com/download/raspberry-pi) for your RPi version. Go for the 64 bit version. **Note that as of 7/30/2020, the link for 18.04 64bit for rpi4 is broken, and points to the rpi3 version. You can grab the correct rpi4 version from [this direct link](https://ubuntu.com/download/raspberry-pi/thank-you?version=18.04&versionPatch=.4&architecture=arm64+raspi4)**.
+
+Follow the instructions on the download page for preparing the image for the RPi. Namely:
+
+- Flash Ubuntu onto your microSD card. There are instructions for doing this on [Ubuntu](https://ubuntu.com/tutorials/create-an-ubuntu-image-for-a-raspberry-pi-on-ubuntu#1-overview), [Windows](https://ubuntu.com/tutorials/create-an-ubuntu-image-for-a-raspberry-pi-on-windows), and [Mac](https://ubuntu.com/tutorials/create-an-ubuntu-image-for-a-raspberry-pi-on-macos)
+- Attach a monitor and keyboard to the RPi
+- Insert the flashed SD card in the RPi
+- Power it on
+- Login, using "ubuntu" for the username and password
+
+You should now be logged in to your newly minted copy of Ubuntu 18.04 server!
+
+You probably will also want to connect to your newly configured RPi remotely over ssh, rather than having to using a separate monitor every time. Follow these steps.
+
+1. Connect to wifi from the command line
+    1. Instructions [here](https://linuxconfig.org/ubuntu-20-04-connect-to-wifi-from-command-line) (basically, you need to edit the `/etc/netplan/50-cloud-init.yaml` file and add your wifi network)
+    2. `SSID-NAME-HERE` is your network's name, and `PASSWORD-HERE` is the password for it.
+    3. After following these steps, you should see an ip address assigned in the output of `ip a`. It will be an `inet` value like `192.168.1.18`, underneath an interface entry like `wlan0`
+2. Enable SSH
+    1. Instructions [here]([https://askubuntu.com/a/681768). Namely, run `sudo systemctl enable ssh.socket` from the command line
+    3. Now you should be able to login from your dev machine. `ssh ubunut@192.168.1.18`, or whatever the ip address of your RPi is.
+    4. It should prompt you for a password. Once you enter it successfully, you'll be logged! The `enable` step above should configure the ssh server to automatically come up on reboot, so you can just login to the RPi remotely from now on.
+
+<!-- 3. Add your host/development machine's SSH key to the RPi `authorized_keys` file.
```

Comment:
I'd just leave this out, or refer to a source for this. Not essential imo

Reply:
Just leaving it, as it's already written, isn't too cluttery, and the potential upside of hand-holding for those less familiar with SSH outweighs the downside of possible wordiness.